engines/bastion_katello/test/content-hosts/bulk/content-hosts-bulk-action-subscriptions.controller.test.js

```diff
@@ -28,9 +28,10 @@ describe('Controller: ContentHostsBulkActionSubscriptionsController', function()
         CurrentOrganization = 'foo';
     });
 
-    beforeEach(inject(function($controller, $rootScope, $q) {
+    beforeEach(inject(function($injector, $controller, $rootScope, $q) {
         $_q_ = $q;
         $scope = $rootScope.$new();
+        GlobalNotification = $injector.get('GlobalNotification');
```

Comment:
This is minor but feel free to change if you want when you fix the build.  You can just inject like this as well:

```javascript
beforeEach(inject(function(_GlobalNotification_, $controller, $rootScope, $q) {
          $_q_ = $q;
          $scope = $rootScope.$new();
         GlobalNotification = _GlobalNotification_;
...
```